README.md

```diff
@@ -2,7 +2,7 @@
 
 Home Assistant Operating System (formerly HassOS) is an operating system optimized for hosting [Home Assistant](https://www.home-assistant.io) and its [Add-ons](https://www.home-assistant.io/addons/).
 
-Home Assistant Operating System uses Docker as Container engine. It by default deploys the Home Assistant Supervisor as a container. Home Assistant Supervisor in turn uses the Docker container engine to control Home Assistant Core and Add-Ons in separate containers. Home Assistant Operating System is **not** based on a regular Linux distribution like Ubuntu. It is built using [buildroot](https://buildroot.org/) and it is optimized for running Home Assistant, especially on single board compute (SBC) devices like the Pi, ODROID, NUC and Tinker Board (see supported hardware below).
+Home Assistant Operating System uses Docker as Container engine. It by default deploys the Home Assistant Supervisor as a container. Home Assistant Supervisor in turn uses the Docker container engine to control Home Assistant Core and Add-Ons in separate containers. Home Assistant Operating System is **not** based on a regular Linux distribution like Ubuntu. It is built using [buildroot](https://buildroot.org/) and it is optimized for running Home Assistant, especially on single board compute (SBC) devices like the Pi, ODROID, NUC, and Tinker Board (see supported hardware below).
```

Comment:
Adding this comma is incorrect. We don't use the oxford comma in the documentation for our project.

See: <https://developers.home-assistant.io/docs/documenting/standards#general-documentation>

Reply:
@frenck Thanks for the review. Make sense, updated in [2c3d53c8c430e9ced0cba84dde98197090039111](https://github.com/home-assistant/operating-system/pull/930/commits/2c3d53c8c430e9ced0cba84dde98197090039111)